CHANGELOG.md

```diff
@@ -5,6 +5,30 @@ _**Note:** This is in reverse chronological order, so newer entries are added to
 
 ## Swift 5.7
 
+* [SE-0353][]:
```

Comment:
Slava added a brief description of SE-0353 in https://github.com/apple/swift/pull/59028; could you move this new description to the existing bullet point?

Reply:
Good catch, updated.